packages/design-system/src/components/contextMenu/menu.js

```diff
@@ -277,7 +303,13 @@ const Menu = ({
   ]);
 
   return (
-    <MenuWrapper isIconMenu={isIconMenu} role="menu">
+    <MenuWrapper
+      ref={wrapperRef}
+      isIconMenu={isIconMenu}
+      role="menu"
+      onBlur={handleBlur}
```

Comment:
Looks like `onBlur` now hooks into `focusout` in react 17+: https://blog.saeloun.com/2021/05/14/react-17-uses-browse-focusin-focusout-for-onfocus-onblur